backend/src/controllers/confirmation.controller.ts

```diff
@@ -2,7 +2,7 @@ import { Order } from './../models/order.model';
 import { Request, Response } from 'express';
 import jwt from 'jsonwebtoken';
 
-function parseJwt(token: string): any {
+const parseJwt = (token: string): any => {
```

Comment:
You can create an interface of payload this JWT. You will use it when creating and when parsing.